.clj-kondo/config.edn

```diff
@@ -1,4 +1,6 @@
-{:linters {:unresolved-symbol
+{:linters {:unresolved-namespace
+           {:exclude [clojure.string]}
```

Comment:
I wasn't sure actually, but it seems it's always required. I tried this: 
```
clj -e '(clojure.string/join "," [1 2])'
"1,2"
```

We can undo this later if we find a negative counter example